index.js

```diff
@@ -28,6 +32,14 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 3", "color: red") 
 
+
+function drinkWater(currentThirstTevel) { 
+  console.log("Man I sure am thirsty")
+  current_thirst_level -= 1
+  console.log("Ahh that hits the spot")
+  current_thirst_level
+}
```

Comment:
This one won't quite work, since you're using a different variable name in a couple places (`currentThirstTevel` vs `current_thirst_level`). You also need to **return** the `currentThirstLevel` at the end:

```js
function drinkWater(currentThirstLevel) { 
  console.log("Man I sure am thirsty")
  currentThirstLevel -= 1
  console.log("Ahh that hits the spot")
  return currentThirstLevel
}
```